setup.cfg

```diff
@@ -1,4 +1,7 @@
 [flake8]
-# 201 and 202 are spaces after ( and before ), 501 is line length. These
-# are exceptions allowed (encouraged?) by Galaxy style guidelines.
-ignore = E201,E202,E501
+# These are exceptions allowed (encouraged?) by Galaxy style guidelines.
+# 128 continuation line under-indented for visual indent
+# 201 and 202 are spaces after ( and before )
+# 203 whitespace before ':'
```

Comment:
@martenson If there is really a closed team meeting on coding style - I will leave the meeting. You guys can make a closed decision - but I don't want to have any part of it. These things should be discussed in the open.